src/component/time-event/time-event-actions.js

```diff
@@ -2,19 +2,45 @@ import axios from 'axios';
 import { getCurrentUserId } from '../login-page/login-selector';
 import { showMessage } from '../message-snackbar/message-actions';
 import EventErrorMessage from './time-event-const';
-import { isTimeTemplateActive } from '../time-template/template-selector';
+import { getTimeTemplateNameById, isTimeTemplateActive } from '../time-template/template-selector';
+
 
 export const SET_ACTIVE_TIME_EVENT = 'SET_ACTIVE_TIME_EVENT';
+export const SET_ACTIVE_TIME_EVENT_DURATION = 'SET_ACTIVE_TIME_EVENT_DURATION';
+
 
 const PEOPLE_TIME_API_PATH = '/time-events';
 const START_TIME_EVENT = '/start';
 const STOP_TIME_EVENT = '/stop';
 axios.defaults.baseURL = process.env.BPM_PEOPLE_TIME_API_URL;
 axios.defaults.headers.common['Content-Type'] = 'application/json';
 
-export const setActiveTimeEvent = timeEvent => ({
+export const setActiveTimeEvent = (timeEvent, templateName) => ({
   type: SET_ACTIVE_TIME_EVENT,
   timeEvent,
+  templateName,
+});
+
+export const setActiveTimeEventDuration = duration => ({
+  type: SET_ACTIVE_TIME_EVENT_DURATION,
+  duration,
+});
+
+export const getLastActiveTime = () => ((dispatch, getState) => {
```

Comment:
This method is not used anywhere. But we should call it once the user gets logged in to display the active time event right away.
The call should happen in login-actions.js in the performLogin method after loading all the time-templates.

Also make sure to write tests for this method. Codecov is failing due to this

Reply:
done